executor/executor_fuchsia.h

```diff
@@ -12,8 +12,10 @@
 
 static void os_init(int argc, char** argv, void* data, size_t data_size)
 {
-	if (syz_mmap((size_t)data, data_size) != ZX_OK)
-		fail("mmap of data segment failed");
+	zx_status_t status = syz_mmap((size_t)data, data_size);
+	if (status != ZX_OK) {
+		fprintf(stderr, "mmap of data segment failed with: %d", status);
```

Comment:
If we fail to map the data address range, nothing good will happen, fuzzing will not do anything meaningful, no syscall that accepts any pointers will ever succeed. With the printf, it will happen completely silently. Nobody will notice and nobody can see this output. Coverage will just drop from the cliff and no notifications will be generated. We need fail here.